airrohr-firmware/DHT.cpp

```diff
@@ -5,17 +5,18 @@ written by Adafruit Industries
 */
 
 #include "DHT.h"
+#include "PCF8574.h"
```

Comment:
@Warukira is this necessary if it was included in DHT.h?

Reply:
@chegejames, I' m done. Pushing the requested changes, let me know if you have any more questions.